### DISCUSSION TITLE
FCP text to SRT converter?

### DISCUSSION THREAD

**Discussion by TomThistlethwaite** [CommandPost/FCPCafe#258] [Final Cut Pro Questions] [answered]
Hey all,

While editingtools.io is great for converting SRT to FCP text, does anyone have a decent workflow for going the other direction? 

The Spherico X-Title Extractor used to be great for this, but now no longer working with current XML version. 

Cheers,

Tom

**Comment by latenitefilms** [comment, marked as answer]
Ummm... good question! You can probably just manually tweak the FCPXML version and still use Spherico X-Title Extractor in most cases?

**Comment by TomThistlethwaite** [comment]
Hey Chris, thanks that worked! I didn't realise that the latest XML format is actually a package / folder and the actual XML file is buried inside. Much appreciated!

**Comment by TomThistlethwaite** [comment]
Just a quick update on this for anyone else, the amazing Matt at editingtools.io has updated their converter so you can now go from FCPX titles to an SRT file. Kudos to those guys for being so responsive. 

https://en.editingtools.io/subtitles/
